jekyll/_cci2/test-config-policies.adoc

```diff
@@ -0,0 +1,613 @@
+---
+description: Open preview of config policy management for CircleCI, learn about testing policies
+contentTags:
+  platform:
+  - Cloud
+---
+= Test config policies
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+NOTE: Config policy management is available on the **Scale** plan and is currently in **open preview**. All aspects of the feature are subject to change.
+
+CAUTION: While config policy management is in open preview stage, it should **not** be used for compliance purposes. The feature might become unavailable occasionally during **open preview**, during which build configurations may **not** be evaluated against policies.
+
+Follow the how-to guides on this page to write and set up tests for your config policies.
+
+This guide assumes you are familiar with writing policies for use with the config policy management feature. If you would like more information or information on configuration policy management in general, or the processes around writing policies, see the xref:config-policy-management-overview.adoc[Config policy management overview] and xref:use-the-cli-and-vcs-for-config-policy-management.adoc[Use the CLI and VCS for config policy management] pages.
+
+[#prerequisites]
+== Prerequisites
+
+The steps in this guide assume you have followed and completed the xref:/docs/use-the-cli-and-vcs-for-config-policy-management/#create-a-policy#[Create a policy] guide. In that guide you will have done the following:
```

Comment:
This testing how to guide should follow on nicely from this section: https://circleci.com/docs/use-the-cli-and-vcs-for-config-policy-management/#create-and-upload-a-policy